psychopy/experiment/components/mouse/__init__.py

```diff
@@ -508,7 +508,7 @@ def writeRoutineEndCode(self, buff):
             # use that set of properties to create set of addData commands
             for property in mouseDataProps:
                 if store == 'every frame' or forceEnd == "never":
-                    code = ("%s.addData('%s.%s', %s.%s)\n" %
+                    code = ("%s.addData('%s.%s', %s.%s[0])\n" %
```

Comment:
This _looks_ like you're only storing the first click now, no? But the code section suggests you should be storing every frame (i.e. many)